packages/address/src/graphqlQuery.ts

```diff
@@ -19,6 +19,9 @@ query countries($locale: SupportedLocale!) {
       postalCode
       zone
     }
+    optionalLabels {
+      address2
```

Comment:
shouldn't this be querying all fields too?

Reply:
It could query everything but for the purposes right now it could be over-fetching. In the countries call, every single country is fetched. My worry was that it would be wasteful or slow things down to have all these fields if only address2 is being used. 

It could be handy though because then you wouldn't have to update this package if you want new optional fields in the future. 

What do you think?

Reply:
my question was more in the likes of "why do we only care about address2 optionality?". IDK how much querying all fields would impact in performance, to be honest (server side wise, I would say it's negligible)

Reply:
I've only included address2 because thats the only field i'd like this functionality for in `web` right now